easybuild/easyconfigs/m/minimap2/minimap2-2.28-GCCcore-13.3.0.eb

```
@@ -0,0 +1,55 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# DeepThought, Flinders University
+# Updated to 2.22
+# R.QIAO
+
+# Update Petr Král (INUITS)
+easyblock = 'MakeCp'
+
+name = 'minimap2'
+version = '2.28'
+
+homepage = 'https://github.com/lh3/minimap2'
+description = """Minimap2 is a fast sequence mapping and alignment
+program that can find overlaps between long noisy reads, or map long
+reads or their assemblies to a reference genome optionally with detailed
+alignment (i.e. CIGAR). At present, it works efficiently with query
+sequences from a few kilobases to ~100 megabases in length at an error
+rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
+test data sets, minimap2 is over 20 times faster than most other
+long-read aligners. It will replace BWA-MEM for long reads and contig
+alignment."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/lh3/%(name)s/releases/download/v%(version)s/']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['ffa5712735d229119f8c05722a0638ae0cc15aeb8938e29a3e52d5da5c92a0b4']
+
+builddependencies = [('binutils', '2.42')]
+
+dependencies = [('zlib', '1.3.1')]
+
+buildopts = 'CC="${CC}" CFLAGS="${CFLAGS}" INCLUDES="${CPPFLAGS}"'
```

```suggestion
buildopts = 'CC="${CC}" CFLAGS="${CFLAGS}" CPPFLAGS="${CPPFLAGS}"'
```
Ok I went down an ARM rabit whole and this is breaking the flags set for ARM and I don't know why INCLUDES is being use here and not CPPFLAGS.
@verdurin do you still remember why you had done this?